recipes/libevent/all/conanfile.py

```diff
@@ -92,14 +90,30 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "cmake"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
-        self.cpp_info.names["cmake_find_package"] = "Libevent"
-        self.cpp_info.names["cmake_find_package_multi"] = "Libevent"
-
-        if self.settings.os == "Linux":
-            self.cpp_info.system_libs.extend(["rt"])
-
+        self.cpp_info.filenames["cmake_find_package"] = "Libevent"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "Libevent"
+        self.cpp_info.names["cmake_find_package"] = "libevent"
+        self.cpp_info.names["cmake_find_package_multi"] = "libevent"
+        # core
+        self.cpp_info.components["core"].names["pkg_config"] = "libevent_core"
+        self.cpp_info.components["core"].libs = ["event_core"]
+        if self.settings.os == "Linux" and not self.options.disable_threads:
+            self.cpp_info.components["core"].system_libs = ["pthread"]
```

Comment:
```suggestion
            self.cpp_info.components["core"].system_libs = ["pthread", "rt"]
```

I see no `rt` listed.

Reply:
I've removed rt, I don't see it while calling `objdump -p` on shared libs.

Reply:
makes sense! thanks!